### DISCUSSION TITLE
SpeedTest

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [termux/termux-app#3521] [Ideas]
Know more: https://www.speedtest.net/apps/cli

Add the package in Termux `speedtest`. It is possible to test internet latency via terminal. A reasonably useful resource.

`pkg install speedtest`

**Comment by sylirre** [comment]
https://www.speedtest.net/apps/cli is a closed-source program distributed under EULA.

Details: https://www.speedtest.net/about/eula

> Restrictions On Use. You shall not: (a) copy the Software, except as expressly permitted herein; (b) modify, translate, adapt, or otherwise create derivative works or improvements, whether or not patentable, of the Software; (c) reverse engineer, disassemble, decompile, decode, or otherwise indirectly or directly attempt to derive or gain access to the source code of the Software or any part thereof; (d) remove, delete, alter, or obscure any trademarks or any copyright, trademark, patent, or other intellectual property or proprietary rights notices from the Software, including any copy thereof; **(e) rent, lease, lend, sell, sublicense, assign, distribute, publish, transfer, or otherwise make available the Software, or any features or functionality of the Software, to any third party for any reason, including by making the Software available on a network where it is capable of being accessed by more than one device at any time**; or (f) install or use the Software on any router, modem, or other non-personal computer device.

We don't package this kind of software.
